dev/NavigationView/NavigationViewItemsFactory.cpp

```diff
@@ -6,6 +6,7 @@
 #include "NavigationViewItemBase.h"
 #include "NavigationViewItem.h"
 #include "ItemTemplateWrapper.h"
+#include <ElementFactoryRecycleArgs.h>
 
 void NavigationViewItemsFactory::UserElementFactory(winrt::IInspectable const& newValue)
 {
```

Comment:
While you are here, @ojhad, I think "automagically" in the comment below should mean "automatically".

Reply:
"Automagically" is the intended word, however you are right, it is very similar in meaning as "automatically"